src/prog_models/linear_model.py

```diff
@@ -1,6 +1,7 @@
 # Copyright © 2021 United States Government as represented by the Administrator of the
 # National Aeronautics and Space Administration.  All Rights Reserved.
 
+from sympy import false, true
```

Comment:
What role does this play? I've never seen sympy true and false variables used. Will the language-included `True` and `False` statements work for your use case?